src/main/java/uk/gov/hmcts/dts/fact/entity/Facility.java

```diff
@@ -32,4 +42,15 @@
         inverseJoinColumns = @JoinColumn(name = "facility_type_id")
     )
     private FacilityType facilityType;
+
+    public Facility(final String name, final String description, final String descriptionCy, FacilityType facilityType) {
```

Comment:
All the other method arguments are declared final. May be we could do the same for facilityType

Reply:
updated with final